includes/class-edd-html-elements.php

```diff
@@ -489,14 +489,14 @@ public function category_dropdown( $name = 'edd_categories', $selected = 0 ) {
 	 *
 	 * @since 1.5.2
 	 *
-	 * @param string $name         Name attribute of the dropdown
-	 * @param int    $selected     Year to select automatically
-	 * @param int    $years_before Number of years before the current year the dropdown should start with
-	 * @param int    $years_after  Number of years after the current year the dropdown should finish at
-	 *
+	 * @param string $name         Name attribute of the dropdown.
+	 * @param int    $selected     Year to select automatically.
+	 * @param int    $years_before Number of years before the current year the dropdown should start with.
+	 * @param int    $years_after  Number of years after the current year the dropdown should finish at.
+	 * @param string $id A unique identifier for the field.
```

Comment:
Can you add some spaces between `$id` and `A` to line up the description with the others?